src/GCRealTimeMon/Configuration/Theme.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace GCRealTimeMon.Configuration
+{
+    /// <summary>
+    /// Constants used for spectre console output theming.
+    /// </summary>
+    internal static class Theme
+    {
+        public static string ToHeader(string headerName) => $"[bold {Constants.GCTableHeaderColor}]{headerName}[/]";
+
+        public static string ToMessage(string message) => $"[{Constants.MessageColor}]{message}[/]";
+
+        public static string ToWarning(string message) => $"[{Constants.WarningColor}]{message}[/]";
+
+        public static string TotalHeap = $"[bold {Constants.TotalHeapColor}]Total Heap:[/]";
+        public static string Gen0Heap = $"[bold {Constants.Gen0HeapColor}]Gen 0:[/]";
+        public static string Gen1Heap = $"[bold {Constants.Gen1HeapColor}]Gen 1:[/]";
+        public static string Gen2Heap = $"[bold {Constants.Gen2HeapColor}]Gen 2:[/]";
+        public static string Gen3Heap = $"[bold {Constants.Gen3HeapColor}]Gen 3:[/]";
+        public static string Gen4Heap = $"[bold {Constants.Gen4HeapColor}]Gen 4:[/]";
+
+        // Future - these could be configurable via yaml
+        // Color reference: https://spectreconsole.net/appendix/colors
+        internal class Constants
```

Comment:
NIT: But can this class be made a ``static`` class?

Reply:
Yes!